internal/sdkv2provider/schema_cloudflare_teams_list.go

```diff
@@ -34,10 +34,34 @@ func resourceCloudflareTeamsListSchema() map[string]*schema.Schema {
 		"items": {
 			Type:        schema.TypeSet,
 			Optional:    true,
+			MaxItems:    1,
```

Comment:
this breaks current tests; is that expected?

Reply:
Just pushed a fix changing it. That was accidentally left out.